src/Numerics/DGmethods/DGmodel_kernels.jl

```diff
@@ -557,6 +616,13 @@ Computational kernel: Evaluate the surface integrals on right-hand side of a
     @inbounds e[1] = elems[eI]
 
     @inbounds for f in faces
+        # The remainder model needs to know which direction of face the model is
+        # being evaluated for. So faces 1:(nface - 2) are flagged as
+        # `HorizontalDirection()` faces and the remaining two faces are
+        # `VerticalDirection()` faces
+        face_direction =
```

Comment:
Asking for curiosity: Is this convention documented anywhere (enumerating horizontal side-walls as 1--4 and faces 5--6 as the top/bottom faces)?

Reply:
Not sure.